src/libraries/System.Security.Cryptography/tests/HKDFTests.cs

```diff
@@ -245,11 +261,18 @@ public static IEnumerable<object[]> GetPrkTooShortTestCases()
             {
                 yield return new object[] { HashAlgorithmName.MD5, 128 / 8 - 1 };
             }
+
+            if (PlatformDetection.SupportsSha3)
+            {
+                yield return new object[] { HashAlgorithmName.SHA3_256, SHA3_256.HashSizeInBytes - 1 };
+                yield return new object[] { HashAlgorithmName.SHA3_384, SHA3_384.HashSizeInBytes - 1 };
+                yield return new object[] { HashAlgorithmName.SHA3_512, SHA3_512.HashSizeInBytes - 1 };
+            }
         }
 
-        private static Rfc5869TestCase[] Rfc5869TestCases { get; } = new Rfc5869TestCase[7]
+        private static HkdfTestCase[] HkdfTestCases { get; } = new HkdfTestCase[7]
```

Comment:
If these are still all from RFC5869, it's probably good to say that somewhere.  (A comment, or that this property could still say Rfc5869, but the driver method pulls from here and the SHA3 custom extensions)